### ISSUE TITLE
BibTeX output does not include name-particle

### ISSUE THREAD

**Issue by johan12345** [citation-file-format/ruby-cff#53]
It seems that the `name-particle` is not included into the `author` field when using the `BibtexFormatter`. For a person named
```yaml
given-names: Ludwig
family-names: Beethoven
name-particle: van
```
I would expect the BibTeX output to be
```
author = {van Beethoven, Ludwig}
```
(which is the correct format understood by BibTeX, as seen on the last page of [these docs](https://mirrors.dotsrc.org/ctan/biblio/bibtex/base/btxdoc.pdf)), but it is just
```
author = {Beethoven, Ludwig}
```
(see also [this real-world example](https://github.com/johan12345/gcs_python) on GitHub, where the APA format does include the particle, but the BibTeX format doesn't).

**Comment by hainesr** [comment]
Thanks for catching this @johan12345. Hopefully now fixed here on HEAD and will percolate through to GitHub in time.

**Comment by johan12345** [comment]
That was quick, thanks! :)

**Comment by arfon** [comment]
FYI I'm planning on updating the GitHub gem in the next week or so (currently away from my work machine so can't access the GitHub network).

**Comment by hainesr** [comment]
It would be nice to get the `1.2.0` schema published and then get the ruby-cff version 1.0 out for this. Not sure if time scales will allow!
